tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "wipefs --all --force {{ item }}"
+  register: signatures_before
+  changed_when: '"offset" in signatures_before.stdout'
+  with_items: "{{ disk_wipe_disks }}"
+
+- name: Re-read partion tables
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "partprobe {{ item }}"
+  changed_when: false
+  with_items: "{{ disk_wipe_disks }}"
+
+- name: Erase the disk data
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "shred -v -n 1 --random-source=/dev/zero -s 200M {{ item }}"
```

Comment:
The **200M** value here is hard-coded.  This should probably be user-configurable, as the **200M** value was specific to a single application we were testing, and there may be others that require a different value.

Reply:
updated to use variable